src/main/java/ch/njol/skript/command/Commands.java

```diff
@@ -303,30 +271,25 @@ public static void registerListeners() {
 
 			Bukkit.getPluginManager().registerEvents(new Listener() {
 				@EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
-				public void onPlayerChat(final AsyncPlayerChatEvent e) {
-					if (!SkriptConfig.enableEffectCommands.value() || !e.getMessage().startsWith(SkriptConfig.effectCommandToken.value()))
+				public void onPlayerChat(AsyncPlayerChatEvent event) {
+					if (!SkriptConfig.enableEffectCommands.value() || !event.getMessage().startsWith(SkriptConfig.effectCommandToken.value()))
 						return;
-					if (!e.isAsynchronous()) {
-						if (handleEffectCommand(e.getPlayer(), e.getMessage()))
-							e.setCancelled(true);
+					if (!event.isAsynchronous()) {
+						if (handleEffectCommand(event.getPlayer(), event.getMessage()))
+							event.setCancelled(true);
 					} else {
-						final Future<Boolean> f = Bukkit.getScheduler().callSyncMethod(Skript.getInstance(), new Callable<Boolean>() {
-							@Override
-							public Boolean call() throws Exception {
-								return handleEffectCommand(e.getPlayer(), e.getMessage());
-							}
-						});
+						Future<Boolean> f = Bukkit.getScheduler().callSyncMethod(Skript.getInstance(), () -> handleEffectCommand(event.getPlayer(), event.getMessage()));
```

Comment:
This seems to replicate the behavior of the Task.callSync method. Might be worth using it here?